Rubberduck.VBEEditor/SafeComWrappers/VB6/VBComponents.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Rubberduck.VBEditor.SafeComWrappers.VB6
 {
-    public class VBComponents : SafeComWrapper<VB.VBComponents>, IVBComponents
+    public class VBComponents : SafeEventedComWrapper<VB.VBComponents, VB._dispVBComponentsEvents>, IVBComponents
```

Comment:
I appreciate we're not supporting VB6 yet, but...  as we're not implementing the event interfaces for the VB6 versions here yet, I think it would be better to revert this change.  Anyway, I think the VB._dispVBProjectsEvents is not the correct interface in VB6 as it uses different, but very similar, source interfaces.    Maybe revert and add a TODO